cnames_active.js

```diff
@@ -1566,6 +1566,7 @@ var cnames_active = {
   "kolorfield": "kodie.github.io/kolorfield",
   "komada": "dirigeants.github.io/komada",
   "komo": "jskull.github.io/komo",
+  "komps": "bemky.github.io/komps/",
```

Comment:
```suggestion
  "komps": "bemky.github.io/komps",
```
As the CI error notes, no trailing slash needed :)